### PR TITLE
Update: Changelog

### PR DESCRIPTION
**Pull request by Jitendra2603**
- Added an image showcasing the new API key features
- Added the updated info on API key features

### CODE REVIEW

File: changelog/product-updates.mdx [hunk 1]
```diff
@@ -4,6 +4,21 @@ description: "New updates and improvements"
 mode: "wide"
 ---
 
+<Update label="December 3, 2024">
+
+<Frame>![API Keys](/changelog/images/changelog-03-12-24.webp)</Frame>
```
Comment (louisjoecodes) [inline changelog/product-updates.mdx:9]: Image is low quality and doesn't add much so would remove

File: changelog/product-updates.mdx [hunk 2]
```diff
@@ -4,6 +4,21 @@ description: "New updates and improvements"
...
+
+## API Key Updates
+
+- **Credit Usage Limits**: Set specific credit limits for API keys to control costs and manage usage across different use cases. [Check out](https://elevenlabs.io/app/settings/api-keys)
+- **Workspace API Keys**: Enhanced with access permission controls matching user API key capabilities
```
Comment (louisjoecodes) [inline changelog/product-updates.mdx:14]: This line isn't specific enough - don't understand it

**Review by louisjoecodes — CHANGES_REQUESTED** [review]
Thanks! Added comments

File: changelog/product-updates.mdx (hunk 1 above)
Comment (Jitendra2603) [reply changelog/product-updates.mdx:9]: I think none of the images add much, was just following your format, I can add this HQ old one if you want: https://mintlify.s3.us-west-1.amazonaws.com/elevenlabs-docs/api-reference/images/api_key.webp

File: changelog/product-updates.mdx (hunk 2 above)
Comment (Jitendra2603) [reply changelog/product-updates.mdx:14]: Thoughts on:
- **Credit Usage Limits**: Set specific credit limits for API keys to control costs and manage usage across different use cases by setting "Access"/"No Access" to features like Dubbing, Audio Native, and more. [Check out](https://elevenlabs.io/app/settings/api-keys)
- **Workspace API Keys**: Enhanced with access permission controls matching user API key capabilities like "Read"/"Read and Write" Access to User, Workspace, and History.

Comment (louisjoecodes) [reply changelog/product-updates.mdx:14]: Maybe this is better for WS Api keys: 
Workspace API Keys: Now support access permissions, such as "Read" or "Read and Write," for User, Workspace, and History resources.

File: changelog/product-updates.mdx (hunk 1 above)
Comment (louisjoecodes) [reply changelog/product-updates.mdx:9]: Then it wouldn't be up to date with the latest functionality right?

File: changelog/product-updates.mdx (hunk 2 above)
Comment (Jitendra2603) [reply changelog/product-updates.mdx]: Added, thanks!

**Comment by louisjoecodes** [comment]
That's the old api view in the image

**Pull request merged**